sdk/core/azure-core/azure/core/polling/async_base_polling.py

```diff
@@ -152,12 +154,26 @@ async def request_status(self, status_link: str) -> PipelineResponse[HttpRequest
         """
         if self._path_format_arguments:
             status_link = self._client.format_url(status_link, **self._path_format_arguments)
+        request_params: Dict[str, Any] = {}
+        if self._lro_options and "api_version" in self._lro_options:
```

Comment:
This code is not async and therefore should be a helper on `_SansIOLROBasePolling` to avoid duplication of code.

Reply:
Updated